scripts/zones/Promyvion-Mea/npcs/qm1.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------
+-- Area: Promyvion Mea
+-- ??? map acquisition
+-----------------------------------
+local ID = require("scripts/zones/Promyvion-Mea/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+-----------------------------------
+
+function onTrigger(player, npc)
+    player:messageSpecial(ID.text.NOTHING_OUT_OF_ORDINARY)
+end
+
+function onTrade(player, npc, trade)
+    if npcUtil.tradeHas(trade, 1722) and not player:hasKeyItem(tpz.ki.MAP_OF_PROMYVION_MEA) then
+        npcUtil.giveKeyItem(player, tpz.ki.MAP_OF_PROMYVION_MEA)
+        player:confirmTrade()
```

Comment:
I know you're just using what's currently in these files, but while we're here, we might as well fix a retail inaccuracy~!

These should be starting an event where the screen flashes and there's text about blinding white lights and images being burned into your mind. This is event 49 for all three promyvions.

(And then give the KI and confirm the trade in onEventFinish for those events)

Reply:
Fixed